hnn_core/network.py

```diff
@@ -349,10 +350,16 @@ def add_evoked_drive(self, name, *, mu, sigma, numspikes,
             Standard deviation of event time distribution
         numspikes : int
             Number of spikes at each target cell
-        sync_within_trial : bool
-            If True, the target cells receive each numspike synchronously. By
-            default (False), spike times arriving at each target cell are
-            sampled independently using the Gaussian parameteres (mu, sigma).
+        n_drive_cells : int | 'n_cells'
+            The number of drive cells (i.e., ArtificialCell objects) that each
+            contribute an independently sampled synaptic spike to the network
+            according to the Gaussian time distribution (mu, sigma). If
+            'n_cells' (default), a source gets assigned to each of the
+            self.n_cells simulated cells in the network with 1-to-1
```

Comment:
```suggestion
            simulated cells in the network with 1-to-1
```

I just want to prevent exposing too many internals and getting tied to them if users start relying on them.